hnn_core/optimization/objective_functions.py

```diff
@@ -53,7 +53,79 @@ def _rmse_evoked(initial_net, initial_params, set_params, predicted_params,
     if smooth_window_len is not None:
         dpl.smooth(smooth_window_len)
 
-    obj = _rmse(dpl, target, tstop=tstop)
+    obj = _rmse(dpl, kwargs['target'], tstop=tstop)
+
+    obj_values.append(obj)
+
+    return obj
+
+
+def _maximize_psd(initial_net, initial_params, set_params, predicted_params,
+                  update_params, obj_values, scale_factor, smooth_window_len,
+                  tstop, **kwargs):
+    """The objective function for evoked responses.
+
+    Parameters
+    ----------
+    initial_net : instance of Network
+        The network object.
+    initial_params : dict
+        Keys are parameter names, values are initial parameters.
+    set_params : func
+        User-defined function that sets network drives and parameters.
+    predicted_params : list
+        Parameters selected by the optimizer.
+    update_params : func
+        Function to update params.
+    scale_factor : float
+        The dipole scale factor.
+    smooth_window_len : float
+        The smooth window length.
+    tstop : float
+        The simulated dipole's duration.
+    f_bands : list of tuples
+        Lower and higher limit for each frequency band.
+    relative_bandpower : tuple
+        Weight for each frequency band.
+
+    Returns
+    -------
+    obj : float
+        Normalized RMSE between recorded and simulated dipole.
+    """
```

Comment:
This objective function is less elementary than `_rmse_evoked` so you should provide a "Notes" section to the docstring that that briefly describes how this one works and when it should be used.